pkg/tag/tag_test.go

```diff
@@ -45,6 +45,70 @@ func TestSplitTag(t *testing.T) {
 
 }
 
+func TestRegisterCustom(t *testing.T) {
+
+	t.Run("Add new tag", func(t *testing.T) {
+		// Given a certain tag does not exist
+		_, err := FindByName("TestTag")
+		if err == nil {
+			t.Fatalf("expected TestTag to not exist")
+		}
+
+		// When a new tag is registered
+		TagTestTag := Tag{Group: 0x0063, Element: 0x0020}
+		RegisterCustom(Info{
+			Tag:  TagTestTag,
+			VR:   "UT",
+			Name: "TestTag",
+			VM:   "1",
+		})
+
+		// Then the tag is now part of the tag collection
+		_, err = FindByName("TestTag")
+		if err != nil {
+			t.Fatalf("expected TestTag to be accessible with FindByName")
+		}
+		info, err := Find(TagTestTag)
+		if err != nil {
+			t.Fatalf("expected TestTag to be accessible with Find")
+		}
+		if info.VR != "UT" ||
+			info.Name != "TestTag" ||
+			info.VM != "1" {
+			t.Fatal("info of new registered tag is wrong")
```

Comment:
nit: prefer `t.Errorf` here and above to collect all test errors. Typically I use t.Errorf for test failure conditions, and Fatalf for test setup issues where the test needs to stop immediately.